pages/common/supervisorctl.md

```diff
@@ -0,0 +1,24 @@
+# supervisorctl
+
+> Supervisor is a client/server system that allows its users to control a number of processes on UNIX-like operating systems.
+> Supervisorctl command-line client piece of the supervisor which provides a shell-like interface.
+
+- Start/stop/restart a process:
+
+`supervisorctl {{action}} {{process name}}`
+
+- Start/stop/restart all processes in a group:
+
+`supervisorctl {{action}} {{group name}}:*`
+
+- Tail last N **bytes** of process stderr:
```

Comment:
"Tail" is a jargon word in this context -- please use something more generic, e.g. as seen in the tail.md page.